projects/plugins/jetpack/extensions/blocks/sharing-buttons/available-services.js

```diff
@@ -0,0 +1,13 @@
+export default {
+	print: { label: 'Print', icon: 'print' },
+	email: { label: 'Email', icon: 'mail' },
+	facebook: { label: 'Facebook', icon: 'facebook' },
+	linkedin: { label: 'LinkedIn', icon: 'linkedin' },
+	reddit: { label: 'Reddit', icon: 'reddit' },
+	x: { label: 'X', icon: 'x' },
+	tumblr: { label: 'Tumblr', icon: 'tumblr' },
+	pinterest: { label: 'Pinterest', icon: 'pinterest' },
+	pocket: { label: 'Pocket', icon: 'pocket' },
+	telegram: { label: 'Telegram', icon: 'telegram' },
+	'jetpack-whatsapp': { label: 'WhatsApp', icon: 'whatsapp' },
```

Comment:
Where's "Press this"?

<img width="134" alt="image" src="https://github.com/Automattic/jetpack/assets/87168/6e20cbf1-bc4c-4cca-a8e2-b52145f8e0cf">